packages/angular_devkit/core/src/index.ts

```diff
@@ -12,7 +12,7 @@ import * as json from './json/index';
 import * as logging from './logger/index';
 import * as workspaces from './workspace';
 
-export * from './exception/exception';
+export * from './exception/index';
```

Comment:
nit: Should we get rid of the index file?  It appears to only reexport everything from `exception`.